keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -790,13 +790,44 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+A rollout may fail when at least one of the following components are too old because this KEP introduces the new Kubernetes API field:
```

Comment:
First, a comment on the affect of this enabling/disabling this on running Pods (I can't comment on the unchanged lines above). In the answers above, you say that the permission may change. Under what conditions? If the container restarts? Or just if a Pod is recreated?

Reply:
> Under what conditions? If the container restarts? Or just if a Pod is recreated?

The permission(process identities) may change only when
- the pod is set `SupplementalGroupsPolicy: "Strict"`, and
- it is recreated.